roles/maintenance_10_linux/tasks/main.yml

```diff
@@ -198,3 +198,26 @@
   when: "'postfix' in ansible_facts.packages"
   changed_when: "task.rc == 2"  # rc == 2 => file doesn't exist
   failed_when: "task.rc == 1"  # rc == 1 => postalias failed
+
+- <<: *task
+  vars:
+    taskid: 10-067
+    name: "Netboxinventory: Gather pkg list and output to json"
+  ansible.builtin.package_facts:
+    manager: auto
+
+- <<: *task
+  vars:
+    taskid: 10-067
+    name: "Netboxinventory: Gather pkg list and output to json"
+  ansible.builtin.lineinfile:
+    create: yes
+    mode: 0744
+    path: "host_packages_collection-{{ ansible_date_time.date | string }}.txt"
+    insertafter: EOF
+    line: |
```

Comment:
Shouldn't this be `>-` if you want to insert a single line?

Reply:
Yep, I tested the one in the MR and it works(only puts down a single line), but you are right, we should be replacing those line breaks